src/common/reports.ts

```diff
@@ -0,0 +1,10 @@
+import { CalendarDate } from './CalenderDate';
+
+export const getReportQuarter = (start?: CalendarDate, end?: CalendarDate) => {
+  if (!start || !end) return 'Unknown';
```

Comment:
I'd keep this next to the `ReportLabel` component and match it's signature. It's pretty specific to PeriodicReports, so I don't think it belongs in common.
```suggestion
export const getReportLabel = ({ start, end }: Report) => {
  if (!start || !end) return null;
```
Report above coming from https://github.com/SeedCompany/cord-field/blob/ecd9f2ccdb23b384b643371139943672694e0d92/src/components/PeriodicReports/ReportLabel.tsx#L5-L5